library/core/src/time.rs

```diff
@@ -48,6 +48,16 @@ const MICROS_PER_SEC: u64 = 1_000_000;
 ///
 /// let ten_millis = Duration::from_millis(10);
 /// ```
+///
+/// # Formatting `Duration` values
+///
+/// `Duration` intentionally does not have a `Display` impl, as there are a
+/// variety of ways to format spans of time for human readability. `Duration`
+/// provides a `Debug` impl that shows the full precision of the value. The
+/// `Debug` output uses the Unicode "µs" suffix for microseconds; if your
```

Comment:
Why is a semicolon here?

Reply:
The first half of the sentence is related to the Debug impl, which makes it related to the rest of the paragraph; the second, connected thought expands on the implications of the first half.

Reply:
But should we move it to a different paragraphs since that text is already a bit long. It is still connected but in another paragraph.

Reply:
Alright. Done.